login_test.py

```diff
@@ -35,17 +35,28 @@ def test_canResetPassword(dummy_user):
     global dummy_user_email, dummy_user_password
     for driver_constructor in driver_constructor_generator():
         with driver_constructor() as driver:
-            request_password_reset(driver, dummy_user_email)
+            navigate_to_page("login", driver)

```

Comment:
I would prefer to still have `request_password_reset`, `reset_password(token, new_password)` functions (for example). It's nice to have tests written in as high-level language as possible.

Reply:
This should be addressed in the Login Test pull request, as mentioned there.